### ISSUE TITLE
Close button stops working after device is rotated

### ISSUE THREAD

**Issue by rkreutzer** [capacitor-community/photoviewer#50]
**Describe the bug**
While displaying the viewer page, once the device is rotated, even if rotated back to the original position, the close function no longer works. 

**To Reproduce**
1. Display a single image in the viewer
2. Rotate the device
3. Click the close icon

**Expected behavior**
The close button and swipe-to-close should work in any device orientation.

**Smartphone (please complete the following information):**
 - Device: LG G6, Pixel 5
 - OS: Android 9-12
 - Photoviewer Version 3.0.0

**Comment by jepiqueau** [comment]
@rkreutzer recognized as a bug will look at it

**Comment by jepiqueau** [comment]
@rkreutzer Seems fixed in release 3.0.3 when it will be released can you test it and come back with your feedback

**Comment by jepiqueau** [comment]
@rkreutzer Is that fix now on your side?

**Comment by jepiqueau** [comment]
@rkreutzer This has been reproduce so it will be fixed in the next release if you want to test it on your side now you comment this
```
  /*
      override fun onConfigurationChanged(newConfig: Configuration) {
         val view: View = initializeView()
          mContainer?.addView(view)


        super.onConfigurationChanged(newConfig)
    }

   */
```
in imageFragment and in mainFragment can you revert to me if it works.

**Comment by rkreutzer** [comment]
This fixes the issue. Thanks.

**Comment by rkreutzer** [comment]
This fix was not included in 3.0.3. I'm testing using 3.0.4 and it still has the issue.

**Comment by camilocalvo** [comment]
Same here. If the fix is not in, could you please share any workarounds?
